presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -38,6 +40,30 @@ public ServerConfig setCoordinator(boolean coordinator)
         return this;
     }
 
+    public boolean isEnabledUIonSecondaryPort()
+    {
+        return enabledUIonSecondaryPort;
+    }
+
+    @Config("http-server.ui.secondary.port.enabled")
```

Comment:
Can we deduce this property instead of reading it from config? If the ui.http.port is configured then this must be true, otherwise false.